src/pages/homepage/HomePage.js

```diff
@@ -0,0 +1,91 @@
+import styled from "styled-components";
+import place from "./images/place.png";
+import volenteer from "./images/voloenteer.png";
+// import { useState } from "react";
+import useAuthContext from "./hooks/useAuthContext";
+// Styled Components
+const Header = styled.header`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 2rem auto;
+`;
+
+const WelcomeContent = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const Selection = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin: 2rem auto;
+  cursor: pointer;
+`;
+
+const LogIn = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+function HomePage() {
+  console.log("useAuthContext: ", useAuthContext());
+  const { role, setRole, dummyUser } = useAuthContext();
+  // setRole("place");
+  console.log("role: ", role);
+  console.log("dummyUser: ", dummyUser);
+
+  // // State for the role selection - volenteer or place
+  // const [role, setRole] = useState("");
+
+  // const selectionHandler = (role) => {
+  //   setRole(role);
+  // };
+  return (
+    <div>
+      <Header>
+        <h1>
+          MENT<span style={{ color: "purple" }}>OO</span>
```

Comment:
we are using styled components , u can change iniline styling to styled component